pkg/ui/markdown/renderer.go

```diff
@@ -82,52 +128,24 @@ func (r *Renderer) RenderError(title, details, suggestion string) (string, error
 	var content string
 
 	if title != "" {
-		content += fmt.Sprintf("\n# %s\n\n", title)
+		content += fmt.Sprintf("\n# %s\n", title)
 	}
 
 	if details != "" {
-		content += fmt.Sprintf("%s\n\n", details)
+		content += fmt.Sprintf("%s", details)
 	}
 
 	if suggestion != "" {
 		if strings.HasPrefix(suggestion, "http") {
-			content += fmt.Sprintf("For more information, refer to the **docs**\n%s\n", suggestion)
+			content += fmt.Sprintf("\nFor more information, refer to the [docs](%s)", suggestion)
 		} else {
 			content += suggestion
 		}
 	}
-
-	rendered, err := r.Render(content)
-	if err != nil {
-		return "", err
-	}
-
-	// Remove duplicate URLs and trailing newlines
-	lines := strings.Split(rendered, "\n")
-	var result []string
-	seenURL := false
-
-	// Create a purple style
-	purpleStyle := termenv.Style{}.Foreground(r.profile.Color(Purple)).Bold()
-
-	for _, line := range lines {
-		trimmed := strings.TrimSpace(line)
-		if strings.Contains(trimmed, "https://") {
-			if !seenURL {
-				seenURL = true
-				result = append(result, line)
-			}
-		} else if strings.HasPrefix(trimmed, "$") {
-			// Add custom styling for command examples
-			styled := purpleStyle.Styled(strings.TrimSpace(line))
-			result = append(result, " "+styled)
-		} else if trimmed != "" {
-			result = append(result, line)
-		}
+	if !t.IsTerminal(int(os.Stderr.Fd())) {
```

Comment:
We have a `CheckTTYSupport` function that we should use consistently. Note, I think we might have duplicated this function.